hazelcast/include/hazelcast/client/sql/impl/sql_error_code.h

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2008-2021, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+namespace hazelcast {
+namespace client {
+namespace sql {
+namespace impl {
+
+enum class sql_error_code
+{
+    /** Generic error. */
+    GENERIC = -1,
+
+    /** A network connection problem between members, or between a client and a member. */
+    CONNECTION_PROBLEM = 1001,
```

Comment:
Where are others?

Reply:
these are all the codes used. The other values are not directly used. We return int32_t to the user and the `SqlErrorCodes` enum is in `impl` namespace at Java, which is non-public.